build.sbt

```diff
@@ -15,6 +20,6 @@ resolvers +=
   "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
 
 libraryDependencies ++= Seq(
-  "org.scala-js" %%% "scalajs-dom" % "0.8.0",
-  "com.lihaoyi" %%% "upickle" % "0.3.4",
-  "io.surfkit" %%% "scalajs-google-maps" % "0.1-SNAPSHOT")
+  "org.scala-js" %%% "scalajs-dom" % "0.9.2",
+  "com.lihaoyi" %%% "upickle" % "0.6.6",
+  "io.surfkit" %%% "scalajs-google-maps" % "0.0.3-SNAPSHOT")
```

Comment:
Snapshot dependencies dependencies make me a little uncomfortable. I looked into it and it looks like the problem is with 'io.surfkit'. If we decide to continue using this library I will file an issue over there to  make a release. Another option is to do a source dependency.